app/models/fae/static_page.rb

```diff
@@ -30,16 +30,30 @@ def self.setup_dynamic_singleton
 
       fae_fields.each do |name, value|
         type = value.is_a?(Hash) ? value[:type] : value
+        language = value.try(:[], :languages)
```

Comment:
The `try(:[], :languages)` checks to see if the variable is a hash and if said hash has the key `:languages`. It's equivalent to `languages = value[:languages] if value.is_a?(Hash) && value.key?(:languages)`. Which I guess you could write as `languages = value[:languages] if value.is_a?(Hash)`, but I like the sugar of `try`.

I'll update that variable name.